target/linux/ath79/dts/qca9531_comfast_cf-e320n-v2.dts

```diff
@@ -0,0 +1,166 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca953x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "comfast,cf-e320n-v2", "qca,qca9531";
+	model = "COMFAST CF-E320N v2";
+
+	aliases {
+		serial0 = &uart;
+		led-boot = &led_rssihigh;
+		led-failsafe = &led_rssihigh;
+		led-upgrade = &led_rssihigh;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&jtag_disable_pins &led_rssilow_pin &led_rssimediumhigh_pin &led_rssihigh_pin>;
+
+		wan {
+			label = "red:wan";
+			gpios = <&gpio 2 GPIO_ACTIVE_LOW>;
```

Comment:
In ar71xx, wan, lan and wlan LEDs are set up with GPIO_ACTIVE_HIGH. Who is right?

Reply:
Yes, you are right, it should be HIGH based on [old](https://git.openwrt.org/?p=openwrt/openwrt.git;a=commitdiff;h=31952dbd1c40a2cd015ef435bd64fa9b2bb87eec
) commit.

![image](https://user-images.githubusercontent.com/5116363/139599279-c359d383-86bc-46da-a044-4c8240989d09.png)